src/backends/ilp-quoter/index.js

```diff
@@ -61,13 +48,13 @@ class ILPQuoter {
    */
   * getQuote (params) {
     log.debug('Connecting to ' + this.backendUri)
-    const hasPair = yield this.hasPair(params.source_ledger,
+    const hasPair = utils.hasPair(this.pairs, params.source_ledger,
                                        params.destination_ledger)
     if (!hasPair) {
       throw new AssetsNotTradedError('This connector does not support the ' +
         'given asset pair')
     }
-    const currencyPair = lookupCurrencies(params.source_ledger,
+    const currencyPair = utils.getCurrencyPair(params.source_ledger,
```

Comment:
You can probably cleanup utils.hasPair because it's calling utils.getCurrencyPair, and everywhere you use one you're using the other one.

So maybe just have a `utils.hasCurrencyPair` and use `utils.getCurrencyPair`

Is utils.hasPair necessary? If utils.getCurrencyPairs returns an empty pair `[]` then you know that `hasPair` is false, right?